packages/@react-spectrum/image/docs/Image.mdx

```diff
@@ -81,3 +81,27 @@ To internationalize an `Image`, a localized string should be passed to the `alt`
     objectFit="cover" />
 </Flex>
 ```
+
+### Error callback example
+
+```tsx example
+function ImageItem({ src }) {
+  const [isImageMissing, setIsImageMissing] = React.useState(false);
+  const DEFAULT_IMAGE = 'https://771623823872.png';
```

Comment:
I think you meant it to be the same as the Object Fit example above?
`https://i.imgur.com/c3gTKSJ.jpg`

and I think you meant the src down below to be this image, so that we go through the onError

Reply:
I'm sorry i didn't understand your statement clearly. 
In this example, what I'm trying to do is to set a default image in case the image loas fails. `src` is accepted as a prop. if the `src` image url fails to load, the `DEFAULT_IMAGE` url will be set as src.

Reply:
The default image is currently the one that will fail. Src is set to an image which exists.

Reply:
ohh that. i have modified that in the example. please verify.